File: model/user_tokens.go
package model

import (
	"github.com/jinzhu/gorm"
)

type UserTokens struct {
	gorm.Model
	UserID int64  `gorm: "type: bigint; primary_key; not null"`
	Token  string `gorm: "type: tinytext; primary_key; not null"`
}

File: admin/backend/src/db/model/user-tokens.ts
import { Model, snakeCaseMappers } from 'objection';

export class UserTokens extends Model {
    userId: string;
    token: string;
    
    static get tableName() {
        return 'user_tokens';
    }

    static get idColumn() {
        return ['user_id', 'token'];
    }

    static get columnNameMappers() {
        return snakeCaseMappers();
    }
}

export default UserTokens;